Ratings_MVP/Ratings_MVP/MovieDetailsScreen/View/MovieDetailsViewController.swift

```diff
@@ -105,7 +96,15 @@ extension MovieDetailsViewController: UITableViewDataSource {
         ) as? MovieDetailsCell
         else { return UITableViewCell() }
         guard let currentMovieDetail = currnetMovie else { return UITableViewCell() }
-        cell.setCell(movie: currentMovieDetail)
+        cell.setCell(movie: currentMovieDetail, networkService: presenter.networkService)
         return cell
     }
 }
+
+// MARK: - MovieDetailsViewProtocol
+
+extension MovieDetailsViewController: MovieDetailsViewProtocol {
+    func setMovie(movie: Movie?) {
+        currnetMovie = movie
```

Comment:
```suggestion
        currnetMovie = movie
```
если у тебя есть какой-то метод который это использует, вынеси в презенте